core/app.ts

```diff
@@ -179,6 +180,11 @@ function parseArgs(autorestArgs: Array<string>): CommandLineArgs {
       rawValue = join(process.cwd(), rawValue);
     }
 
+    if (rawValue.startsWith('~/')) {
```

Comment:
Feels a little wrong to hardcode this but I guess it's hard to detect that the value is a path.  Perhaps use [`path.resolve`](https://github.com/Azure/autorest/pull/3418) when the value starts with `.` or `~`?

Reply:
`path.resolve` doesn't actually do that.  (tried that first)  It doesn't know `~`